include/ccf/historical_queries_interface.h

```diff
@@ -28,6 +30,27 @@ namespace ccf::historical
       path(p_),
       node_id(n_)
     {}
+
+    void describe_receipt(ccf::Receipt& r)
```

Comment:
Arguably describe(), since this is already a TxReceipt?

Reply:
Sure, this is now called [describe()](https://github.com/microsoft/CCF/pull/2328/files#diff-ddb414d7c3e5af83a2abf8d566044b3f34577fd3428ed6229f2c29d13f0cfbdfR34). I wonder if we should rename one of these types, but haven't found better names. One is the public, serialisable representation of a `Receipt` you can use in your HTTP API, the other is the in-memory representation you can ask for validation. Another approach may be to flatten these, and just write a custom serialiser/API-descriptors for the latter, rather than producing a separate type. Not critical, but I may do this at some point.

Reply:
Yeah I wondered about that too, but equally did not find any good names. I think this is fine for now.